app/models/audited_account.rb

```diff
@@ -0,0 +1,7 @@
+class AuditedAccount < ActiveRecord::Base
```

Comment:
@diogob I think that ProjectAccount is better not? because we handle with address data too :)